prisma/schema.prisma

```diff
@@ -31,6 +31,7 @@ model User {
   description String
   teacherId   String
   phone       String
+  status      String    @default("ACTIVE")
```

Comment:
@Prodip-Kumar-Paul Are we going to have more than 2 statuses? If not then why not use a boolean?

Reply:
yup better approach